AIPscan/worker.py

```diff
@@ -0,0 +1,5 @@
+from AIPscan.application import create_app
+from AIPscan.celery import configure_celery
+
```

Comment:
Done and done!

I do wonder if we might want to remove the `# -*- coding: utf-8 -*-` headers eventually, since we don't intend (at least as far as I know) to support Python 2? That said, they're not doing any harm and I do see the value in consistency so maybe nbd.